UPGRADE.md

```diff
@@ -16,6 +16,11 @@ There is a list of all the repositories maintained by monorepo, changes in log b
 * [shopsys/product-feed-zbozi]
 
 ## [From 7.0.0-alpha4 to Unreleased]
+- make Elasticsearch work
+    - copy new `docker/conf/docker-compose.yml.dist` into your local `docker-compose.yml`

```

Comment:
it isn't as simple as copying the template, I would suggest generalizing it with "check changes in the `docker-compose.yml` template you used and replicate them", just as is written in upgrade from alpha1 to alpha2

Reply:
I would suggest:
    - check changes in the `docker-compose.yml` template you used, a new container with Elasticsearch has to be added

`    - check changes in the ``docker-compose.yml`` template you used, a new container with Elasticsearch has to be added`